### DISCUSSION TITLE
How to make a navigation sidebar? Does anyone have a polished example?

### DISCUSSION THREAD

**Discussion by bjlaur** [zauberzeug/nicegui#3258] [Q&A] [answered]
### Question

I was hoping that someone might have a fleshed out example of how to make a navigation sidebar.

Briefing looking at it, it looks like the best option is to use a combination of left_drawer and q-list/q-item, but I was hoping someone might have a fully polished example to base my work off of rather than trying to do guesswork from scratch.

Just for clarify, here are a few examples of what I am looking for.
Something like this (from simplifi)
<img src="https://github.com/zauberzeug/nicegui/assets/3256354/00e38e9d-25e9-411f-a1b8-9938c3b2d4ad" width="250">
Or this (from US Mobile)
<img src="https://github.com/zauberzeug/nicegui/assets/3256354/97e641fe-ffa6-4c51-a29e-8533cb2014ea" width="350">

**Comment by python-and-novella** [comment, marked as answer]
Maybe Quasar's examples help you.
Layout Gallery | Quasar Framework
https://quasar.dev/layout/gallery

**Comment by bjlaur** [comment]
Thanks @python-and-fiction . That is useful.
